docs/src/use/configure/migration-guide.md

```diff
@@ -674,6 +674,23 @@ You can see the TypeScript types for the flat config file format in the Definite
 
 You can view the type definitions in the [DefinitelyTyped repository on GitHub](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/eslint/index.d.ts).
 
+## Visual Studio Code Support
+
+By default, the Visual Studio Code ESLint plugin does not look for `eslint.config.js` files. To enable support for the new configuration files, edit your `.vscode/settings.json` file and add the following:
```

Comment:
Nowhere else on this page links to the plugin. I think this would be a small nicety for users:

```suggestion
By default, the [Visual Studio Code ESLint plugin](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) does not look for `eslint.config.js` files. To enable support for the new configuration files, edit your `.vscode/settings.json` file and add the following:
```